controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -169,9 +170,47 @@ func (r *ROSAControlPlaneReconciler) reconcileNormal(ctx context.Context, rosaSc
 		}
 	}
 
+	// TODO: token should be read from a secret: https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/4460
+	token := os.Getenv("OCM_TOKEN")
```

Comment:
Can this be implemented before we merge this? Seems like a trivial fix to read the token from a plain secret